sormas-app/app/src/main/java/de/symeda/sormas/app/pathogentest/edit/PathogenTestEditActivity.java

```diff
@@ -0,0 +1,140 @@
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.app.pathogentest.edit;
+
+import android.content.Context;
+import android.os.AsyncTask;
+import android.view.Menu;
+
+import java.util.List;
+
+import de.symeda.sormas.api.utils.ValidationException;
+import de.symeda.sormas.app.BaseActivity;
+import de.symeda.sormas.app.BaseEditActivity;
+import de.symeda.sormas.app.BaseEditFragment;
+import de.symeda.sormas.app.R;
+import de.symeda.sormas.app.backend.common.DaoException;
+import de.symeda.sormas.app.backend.common.DatabaseHelper;
+import de.symeda.sormas.app.backend.sample.PathogenTest;
+import de.symeda.sormas.app.component.menu.PageMenuItem;
+import de.symeda.sormas.app.component.validation.FragmentValidator;
+import de.symeda.sormas.app.core.async.AsyncTaskResult;
+import de.symeda.sormas.app.core.async.SavingAsyncTask;
+import de.symeda.sormas.app.core.async.TaskResultHolder;
+import de.symeda.sormas.app.core.notification.NotificationHelper;
+import de.symeda.sormas.app.person.edit.PersonEditFragment;
+import de.symeda.sormas.app.task.edit.TaskNewActivity;
+import de.symeda.sormas.app.util.Bundler;
+import de.symeda.sormas.app.visit.edit.VisitNewActivity;
+
+import static de.symeda.sormas.app.core.notification.NotificationType.ERROR;
+import static de.symeda.sormas.app.core.notification.NotificationType.WARNING;
+
+public class PathogenTestEditActivity extends BaseEditActivity<PathogenTest> {
+
+    public static final String TAG = PathogenTestEditActivity.class.getSimpleName();
+
+    private AsyncTask saveTask;
+
+    public static void startActivity(Context context, String rootUuid) {
+        BaseActivity.startActivity(context, PathogenTestEditActivity.class, buildBundle(rootUuid));
+    }
+
+
+
+    @Override
+    protected PathogenTest queryRootEntity(String recordUuid) {
+        return DatabaseHelper.getSampleTestDao().queryUuid(recordUuid);
+    }
+
+    @Override
+    protected PathogenTest buildRootEntity() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public Enum getPageStatus() {
+        return null;
+    }
+
+    @Override
+    protected BaseEditFragment buildEditFragment(PageMenuItem menuItem, PathogenTest activityRootData) {
+        return PathogenTestEditFragment.newInstance(activityRootData);
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        super.onCreateOptionsMenu(menu);
+        getSaveMenu().setTitle(R.string.action_save_contact);
```

Comment:
If you copy existing classes to write new ones with similar functionality (which is completely valid), please make sure to rename everything that was semantically associated with the old class (e.g. in this case, it should be R.string.action_save_pathogen_test)